src/Dont/DontClone.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Dont;
+
+use Dont\Exception\NonCloneableObject;
+use Dont\Exception\TypeError;
+
+trait DontClone
```

Comment:
I was under the impression it should be `DontClone` (so more readable when importing the traits:)

```php
namespace Bar;

use Dont\DontClone;

class Foo {
    use DontClone;
}
```
vs
```php
namespace Bar;

use Dont\Clone;

class Foo {
    use Clone;
}
```
(the 2nd is less clear IMO)